src/officehours_api/tests.py

```diff
@@ -163,3 +165,83 @@ def test_meeting_start_notifies_attendees(self, mock_twilio: mock.MagicMock):
             receivers >=
             {'+15555550000', '+15555550001'}
         )
+
+
+class MeetingSerializerTestCase(TestCase):
+    """
+    Test the scenarios where the backend type is invalid, valid,
+    patched to none, and patched to invalid.
```

Comment:
I think we should update the language here. The `test_patch_backend_type_none` is not really about setting the backend type to `None`, it's about updating another attribute and making sure everything still works. This was the bug I found before; we were changing the assignee or the agenda on the meeting, but because the backend type was always validated (we didn't have the `if attrs.get("backend_type")` check), we were accessing a key that didn't exist in the partial data we were passing and getting internal server errors.

```suggestion
    is not updated but something else is, and is updated to be invalid.
```